### PR TITLE
Fix Rights for Escalation

### PR DESCRIPTION
**Pull request by tsmr**
Fix Rights for Escalation

### CODE REVIEW

File: front/ticket.form.php [hunk 1]
```diff
@@ -32,10 +32,7 @@
 
 include('../../../inc/includes.php');
 
-// Same right check as in PluginEscaladeTicket::addToTimeline()
-Session::checkRight(Ticket::$rightname, Ticket::READALL)
-    && Session::checkRight(Ticket::$rightname, Ticket::READASSIGN)
-    && Session::checkRight(Ticket::$rightname, Ticket::CREATE);
+Session::checkLoginUser();
```
Comment (cedric-anne) [inline front/ticket.form.php:35]: It seems a bit dangerous to permit to any connected user to escalate any ticket to a group.

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Review by AdrienClairembault — CHANGES_REQUESTED** [review]
Agreed with @cedric-anne, is there a specific reason why you are removing these rights checks ?

File: front/ticket.form.php (hunk 1 above)
Comment (tsmr) [reply front/ticket.form.php]: Corrected

**Comment by cedric-anne** [comment]
Fixed in #162

**Pull request closed without merging**